packages/dappmanager/src/calls/statsDiskGet.ts

```diff
@@ -0,0 +1,25 @@
+import { HostStatDisk } from "../types";
+import shellExec from "../utils/shell";
+/**
+ * Returns the disk statistics (used, available, etc)
+ */
+export async function statsDiskGet(): Promise<HostStatDisk> {
+  const disk = await shellExec(`df / --block-size=1`);
+  return parseDiskStats(disk);
+}
+
+/**
+ * Parses the 'df /' bash output command
+ * @param disk string with disk usage info
+ */
+function parseDiskStats(disk: string): HostStatDisk {
+  const arr = disk.replace(/\n/g, " ").split(/\s+/);
```

Comment:
`parseDiskStats` should be unit tested like
```ts
it("Should parse df / output", () => {
  const dfOutput = `Filesystem                1K-blocks      Used Available Use% Mounted on
/dev/mapper/mint--vg-root 235782040 184988848  38746448  83% /`;
  const diskStats = parseDiskStats(dfOutput);
  expect(diskStats).to.deep.equal({
    filesystem: "/dev/mapper/mint--vg-root",
    bBlocks: "235782040",
    used: "184988848",
    available: "38746448",
    use: "83%",
    mountedOn: "/",
    useFraction: 0.342153612
  });
});
```

Reply:
I would also improve the parsing algorythm by splitting by rows first
```ts
const [headers, row] = dfOutput.split("\n");
const [filesystem, bBlocks, used, available, use, mountedOn] = row.split(/\s+/);
return {
  filesystem,
  bBlocks,
  used,
  available,
  use,
  mountedOn,
  useFraction: parseInt(used) / (parseInt(used) + parseInt(available))
};

```

Reply:
Im having error with the deep equal test, I gess I should not use it. Since the commands `free `and `df `give different output depending on the host machine.